src/combinators/mod.rs

```diff
@@ -1006,11 +966,11 @@ pub fn tuple<W: Write, List: Tuple<W>>(l: List) -> impl SerializeFn<W> {
 pub fn back_to_the_buffer<W: BackToTheBuffer, Tmp, Gen, Before>(reserved: usize, gen: Gen, before: Before) -> impl SerializeFn<W>
 where Gen: Fn(W) -> GenResult<(W, Tmp)>,
       Before: Fn(W, &Tmp) -> GenResult<W> {
-    move |w: W| {
-        w.reserve_write_use(reserved, &gen, &before).map(|t| t.0)
+    move |w: WriteContext<W>| {
+        W::reserve_write_use(w, reserved, &gen, &before).map(|t| t.0)
     }
 }
-
+*/
```

Comment:
Why is this commented out?

Reply:
Because I had to leave on Friday and wanted to send this out for comment before that. I'll add that back when updating :)